MAINTAINERS.md

```diff
@@ -75,6 +75,15 @@ For PRs related to the Astro Docs website itself, including accessibility (a11y)
 - Conduct any a11y checks or tests you have access to: screen reader behaviour, colour contrast ratios, etc.
 - For most non-trivial PRs, the Docs Infrastructure Lead will likely want to do a final review themselves before the PR is merged.
 
+#### Accessibility
+
+We encourage our maintainers to audit and improve the accessibility of our code base as we strive for inclusion. We have selected a few trusted sources to help you out in your accessibility efforts:
```

Comment:
```suggestion
We encourage our maintainers to audit and improve the accessibility of our code base as we strive for inclusion. We have selected a few trusted sources to help you out in your accessibility efforts when reviewing or making your own PRs:
```
This looks wonderful, @Yan-Thomas ! I only added one little bit to make it explicitly relevant to making/reviewing PRs. Otherwise, LGTM & TYSM!

Reply:
Thanks @sarah11918! :heart: